src/popup.js

```diff
@@ -0,0 +1,206 @@
+const popupCommentInputComment = document.createElement('textarea');
+popupCommentInputComment.id = 'popupCommentInputComment';
+popupCommentInputComment.classList = 'popup-comment-input popup-new-comment-text';
+popupCommentInputComment.cols = '15';
+popupCommentInputComment.rows = '4';
+popupCommentInputComment.placeholder = 'Comment...';
+popupNewComment.appendChild(popupCommentInputComment);
+
+// popup-comment-btn
+export const popupCommentBtn = document.createElement('a');
+popupCommentBtn.id = 'popupCommentBtn';
+popupCommentBtn.classList = 'popup-comment-btn';
+popupCommentBtn.innerHTML = 'Comment';
+popupNewComment.appendChild(popupCommentBtn);
+
+xBtn.addEventListener('click', () => {
+  popup.classList.remove('display');
+  popupCommentList.innerHTML = '';
+});
+
+export function showPopup(nama, image, info) {
+  popupHeaderText.innerHTML = nama;
+  popupImage.src = image;
+  popupDetail.innerHTML = info;
+}
+
+/// get comments
+export const getComments = async (itemId) => {
+  const involvementCommentAPI = `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/v1QM9q7o5iYcOME1s2k2/comments?item_id=${itemId}`;
+  await fetch(involvementCommentAPI).then((response) => response.json()).then((json) => {
+    if (json.length === undefined) {
+      popupCommentHeader.innerHTML = 'Comments (0)';
+      popupCommentList.innerHTML = '';
+    } else {
+      popupCommentHeader.innerHTML = `Comments (${json.length})`;
+      popupCommentList.innerHTML = '';
+      json.forEach((element) => {
+        const newComment = document.createElement('li');
+        newComment.classList = 'popup-comment-item';
+        newComment.innerHTML = ` 
+                                        <label class="popup-comment-author">${element.username}</label>
+                                        <p class="popup-comment-text">${element.comment}</p>
+                                        <p class="popup-comment-date">${element.creation_date}</p>`;
+        popupCommentList.appendChild(newComment);
+      });
+    }
+  }).catch((e) => e);
+};
+
+/// / add new comment
+export const addComment = async (itemId) => {
+  const involvementCommentAPI = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/v1QM9q7o5iYcOME1s2k2/comments';
+  const id = itemId;
+  const name = document.getElementById('popupCommentInputName').value;
+  const comment = document.getElementById('popupCommentInputComment').value;
+  if (name === '' || comment === '') return;
+
+  await fetch(involvementCommentAPI, {
+    method: 'POST',
+    body: JSON.stringify({
+      item_id: id,
+      username: name,
+      comment,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  }).then((response) => {
+    getComments(itemId);
+    document.getElementById('popupCommentInputName').value = '';
+    document.getElementById('popupCommentInputComment').value = '';
+    return response.json();
+  }).catch((e) => e);
+};
```

Comment:
- [ ] [Optional] It'd be better if there appears an error message if the form is not valid.